sections/sections-app/src/webapp/editStudentSections.jsp

```diff
@@ -194,6 +194,12 @@
             });
         </script>
 
+        <f:verbatim><span></f:verbatim>
```

Comment:
```suggestion
        <span>
```

Reply:
I copied this out of what was in the navMenu.jspf, so can make this change to this and the closing tag. Probably something that wasn't updated with the JSF 2 upgrades and should also be cleaned up there. 

Thanks!